libres/lib/analysis/update.cpp

```diff
@@ -283,38 +282,30 @@ load_parameters(enkf_fs_type *target_fs, ensemble_config_type *ensemble_config,
                 meas_data_type *forecast, obs_data_type *obs_data,
                 const local_ministep_type *ministep) {
 
-    int cpu_threads = 4;
-    thread_pool_type *tp = thread_pool_alloc(cpu_threads, false);
-    int matrix_start_size = 250000;
-    int active_ens_size = meas_data_get_active_ens_size(forecast);
-    matrix_type *A = matrix_alloc(matrix_start_size, active_ens_size);
-
-    serialize_info_type *serialize_info = serialize_info_alloc(
-        target_fs, //src_fs - we have already copied the parameters from the src_fs to the target_fs
-        target_fs, ensemble_config, iens_active_index, 0, last_step, A,
-        cpu_threads);
-
     std::unordered_map<std::string, matrix_type *> parameters;
```

Comment:
This no longer needs to be an unordered map, as we are only creating one matrix here.